recipes/dask-gateway/meta.yaml

```diff
@@ -0,0 +1,112 @@
+{% set name = "dask-gateway" %}
+{% set version = "0.2.0" %}
+
+package:
+  version: {{ version }}
+  name: {{ name|lower }}
+
+source:
+  url: https://github.com/jcrist/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 1ae59cc611fb4a7dd1210bdf82e1eef151d450f144ae66d1d478582cd9f95964
+  patches:
+    - setup.py.patch
+
+build:
+  number: 0
+  version: {{ version }}
+  skip: True  # [py<36]
+
+outputs:
+  - name: dask-gateway
+    script: dask-gateway-client.sh
+    build:
+      entry_points:
+        - dask-gateway-scheduler = dask_gateway.dask_cli:scheduler
+        - dask-gateway-worker = dask_gateway.dask_cli:worker
+    requirements:
+      host:
+        - python
+        - pip
+      run:
+        - python
+        - tornado
+        - dask >=2.0.0
+        - distributed >=2.0.0
+    test:
+      imports:
+        - dask_gateway
+      commands:
+        - dask-gateway-scheduler --help
+        - dask-gateway-worker --help
+  - name: dask-gateway-kerberos
+    requirements:
+      - {{ pin_subpackage('dask-gateway', exact=True) }}
+      - pykerberos
```

Comment:
@jcrist @xhochy is `winkerberos` for windows the equivalent of `pykerberos` for osx/linux?

Reply:
Yes

Reply:
Looks like I hardcoded it to pykerberos in the client library though, will patch: https://github.com/jcrist/dask-gateway/blob/34d913de0c6b8fd8da052777a0492e5a0ef63bca/dask-gateway/setup.py#L8

Reply:
https://github.com/jcrist/dask-gateway/pull/60

Reply:
that's fine. usually conda-forge recipes have a `--no-deps` on the pip install command so any dependencies you include in the setup.py are ignored here for conda packaging purposes. But that does mean that your windows wheels would not work :)